COSC-499-W2023/year-long-project-team-1
# feat(user): user account UI

# Welcome to PrivacyPal! 👋

Depends on: #126
Fixes: #91

## Description of the change:

A user dashboard is now available at `/user/dashboard`. Server Actions have been used for these database operations. Login/Logout is also now possible via a Server Action.

<img width="1470" alt="Screenshot 2023-11-21 at 2 56 20 PM" src="https://github.com/COSC-499-W2023/year-long-project-team-1/assets/3927471/7573d8fc-12d8-4936-b09d-acbb486b099a">

## Motivation for the change:
Users should be able to view their personal data and be informed of upcoming appointments. This also served as a chance to integrate Server Actions.

## To Test

1. Ensure the database is set up and running
2. Run Next.js
3. Visit `/user` or `/login` to log in
4. Find the dashboard at `/user/dashboard`
